README.md

```diff
@@ -88,9 +90,11 @@ The following VCF fields can be accessed in the filter expression:
 
  ¹ depends on type specified in VCF header
 
- ² for the usual snpeff and vep annotations, custom types have been specified; any unknown ANN field will simply be of type `str`. If something lacks a custom parser/type, please consider filing an issue in the [issue tracker](https://github.com/vembrane/vembrane/issues).
+ ² if you specify something different for `--annotation-key`, use that key here. For example, for the default VEP `--annotation-key CSQ`, use `CSQ["SYMBOL"]`.
```

Comment:
Really? I would've expected that the variable is always called ANN.

Reply:
Yes, that was what we expected. Testing suggested otherwise. As do the tests:
https://github.com/vembrane/vembrane/blob/1a1e6f8efd80a8606fcacf29ce31269100e74341/tests/testcases/filter/test_ann_csq_annotation_key/config.yaml#L3

And I guess this is where this happens?
https://github.com/vembrane/vembrane/blob/1a1e6f8efd80a8606fcacf29ce31269100e74341/vembrane/representations.py#L38C14-L38C22